apisix/cli/ngx_tpl.lua

```diff
@@ -459,11 +459,11 @@ http {
 
     server {
         {% for _, item in ipairs(node_listen) do %}
-        listen {* item.port *} default_server {% if enable_reuseport then %} reuseport {% end %} {% if item.enable_http2 then %} http2 {% end %};
+        listen {* item.ip *}:{* item.port *} default_server {% if enable_reuseport then %} reuseport {% end %};
```

Comment:
http port also need to enable_http2

Reply:
OK, and it means that the config `apisix.node_listen[]` also has the property `enable_http2` , right?
```yaml
apisix:
  # node_listen: 9080                # APISIX listening port
  node_listen:                   # This style support multiple ports
  - 9080
  # - ip: 127.0.0.2                # Specific IP, If not set, the default value is `0.0.0.0`.
  #   port: 9082
  #   enable_http2: true  
```

Reply:
done

Reply:
> it means that the config apisix.node_listen[] also has the property enable_http2 , right?

Yes.